src/main.rs

```diff
@@ -49,6 +54,8 @@ const EIP_FINALIZER_NAME: &str = "eip.materialize.cloud/destroy";
 const EIP_ALLOCATION_ID_ANNOTATION: &str = "eip.materialize.cloud/allocation_id";
 const EXTERNAL_DNS_TARGET_ANNOTATION: &str = "external-dns.alpha.kubernetes.io/target";
 
+const EIP_QUOTA_CODE: &str = "L-0263D0A3";
```

Comment:
mind adding a comment that says where to find that constant (and others like it, ideally)?

Reply:
:smile: addressed in https://github.com/MaterializeInc/k8s-eip-operator/pull/145/commits/507b3b5aae8770733c105486e8c603ec810fc5a8